ansi/iterm2/file.go

```diff
@@ -0,0 +1,130 @@
+package iterm2
+
+import (
+	"strconv"
+	"strings"
+)
+
+// Auto is a constant that represents the "auto" value.
+const Auto = "auto"
+
+// Cells returns a string that represents the number of cells. This is simply a
+// wrapper around strconv.Itoa.
+func Cells(n int) string {
+	return strconv.Itoa(n)
+}
+
+// Pixels returns a string that represents the number of pixels.
+func Pixels(n int) string {
+	return strconv.Itoa(n) + "px"
+}
+
+// Percent returns a string that represents the percentage.
+func Percent(n int) string {
+	return strconv.Itoa(n) + "%"
+}
+
+// file represents the optional arguments for the iTerm2 Inline Image Protocol.
+//
+// See https://iterm2.com/documentation-images.html
+type file struct {
```

Comment:
maybe call this image instead?

Reply:
The iTerm2 protocol is originally a file transfer protocol. If the `inline` option is disabled or omitted, the file content is then downloaded to the user's Downloads folder

Reply:
oh, interesting 🤔 

thanks for clarifying!